packages/main/src/plugin/dockerode/libpod-dockerode.ts

```diff
@@ -72,9 +72,35 @@ export interface PlayKubeInfo {
   Volumes: { Name: string }[];
 }
 
+export interface PodCreatePortOptions {
+  host_ip: string;
```

Comment:
Throughout podman-desktop I'm finding that we switch between camelCase and snake_case . We should have a consistent style guide of using camelCase OR snake_case. For example, we use `engineId` some places, but here we are using the `host_ip` style.

Reply:
well, the REST API consumes the attributes that we're sending over JSON

if we use `hostIp` it will send `hostIp` in the JSON body while the REST API will expect host_ip

Reply:
I think we use camelCase everywhere except when it's used by the REST endpoints

Reply:
Ahhh, that makes sense. Thanks!